src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,40 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private final DateTimeFormatter dateTimeFormatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder result = new StringBuilder("Report for period ");
+        result.append(getFormattedData(dateFrom, "-", dateTo));
```

Comment:
The method name 'getFormattedData' seems to imply formatting a date, but it's only joining strings with spaces. Consider renaming it to more accurately reflect its functionality, such as 'joinReportData'.